src/components/Card/Card.tsx

```diff
@@ -10,44 +11,38 @@ const addDefaultImg = (e: any) => {
     e.target.src = defaultImageDataURI
 }
 const Card = ({ user }: any) => {
-    let { id, img, name, email, jobTitle, location, links } = user
+    const { id, img, name, email, jobTitle, location, links } = user
```

Comment:
nitpick: we should avoid variable name "img". It should be descriptive.  What image it is?